docs/modules/ROOT/pages/client.adoc

```diff
@@ -141,6 +144,15 @@ quarkus.openapi-generator.codegen.spec.my_spec_yml.type-mappings.DateTime=Instan
 quarkus.openapi-generator.codegen.spec.my_spec_yml.import-mappings.Instant=java.time.Instant
 ----
 
+If you have an object you want to map to an datatype, you can use the schema mappings. For instance, if your schema is based
+on `java.Time.YearMonth`, it will be mapped to an string when generating. If you want to convert the object `YearMonth`then
+you can use the following settings:
```

Comment:
```suggestion
If you have an object you want to map to a datatype, you can use the schema mappings. For instance, if your schema is based on `java.Time.YearMonth`, it will be mapped to a string when generating. If you want to convert the object `YearMonth,` then you can use the following settings:
```

Can you please rephrase this paragraph? I'm not sure if I understood what you meant here.

Reply:
Thanks for the feedback and I understand what you mean, I also had problems understanding want I meant. I tried to rephrase the paragraph but I was never happy about it, but when I read the whole section one more time, I felt that it was enough with an example and description  on how to use it (in the table) and then an link to the documentation on OpenAPI page instead.

It felt more right and easier to understand.

If you feel that this is not the way to go then I will happily try to write an new text :)